### PR TITLE
Adds a solarian coilgun.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
title

### CODE REVIEW

File: code/modules/overmap/ship_weaponry/weaponry/coilgun.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	icon = 'icons/obj/ammo.dmi'
+	icon_state = "trodpack-2"
+	caliber = SHIP_CALIBER_COILGUN
+	overmap_icon_state = "cannon"
+	impact_type = SHIP_AMMO_IMPACT_HE
+
+/obj/item/projectile/ship_ammo/coilgun
+	name = "high-power tungsten rod"
+	icon_state = "heavy"
+	damage = 10000
```
Comment (SleepyGemmy) [inline code/modules/overmap/ship_weaponry/weaponry/coilgun.dm:27]:
```suggestion
	damage = 200
```

File: code/modules/overmap/ship_weaponry/weaponry/coilgun.dm [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	armor_penetration = 1000
```
Comment (SleepyGemmy) [inline code/modules/overmap/ship_weaponry/weaponry/coilgun.dm:28]:
```suggestion
	armor_penetration = 100
```

**Review by SleepyGemmy — APPROVED** [review]

File: code/modules/overmap/ship_weaponry/weaponry/coilgun.dm [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	penetrating = 1
```
Comment (SleepyGemmy) [inline code/modules/overmap/ship_weaponry/weaponry/coilgun.dm:29]:
```suggestion
	penetrating = TRUE
```

**Review by Alberyk — APPROVED** [review]

File: code/modules/overmap/ship_weaponry/weaponry/coilgun.dm (hunk 3 above)
Comment (NonQueueingMatt) [reply code/modules/overmap/ship_weaponry/weaponry/coilgun.dm:29]: it's not a boolean

**Pull request merged**